Dockerfile

```diff
@@ -56,6 +56,16 @@ ENV SPIDERFOOT_LOGS /home/spiderfoot/log
 # Place database and configs outside installation directory
 ENV SPIDERFOOT_DATA /var/lib/spiderfoot
 
+# massdns
+RUN apk --update --no-cache --virtual .build-deps add git build-base go \
+   && git clone --depth=1 https://github.com/blechschmidt/massdns.git \
+   && cd massdns && make && make install
+ENV GOPATH /go
+ENV PATH="$GOPATH/bin:$PATH"
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin"
+RUN GO111MODULE=on go get -v github.com/projectdiscovery/shuffledns/cmd/shuffledns
+RUN apk del .build-deps
+
```

Comment:
There is currently no precedence for installing additional third-party tools in the image. However, this is probably a good idea. Perhaps we should look at also installing other tools such as DNStwist. @smicallef will likely have an opinion here.

Reply:
I tend to agree with @bcoles here. While I'm keen to have SpiderFoot integrate with more recon tools to expand its reach, bundling the binaries or building of all these tools into the SpiderFoot Docker image feels heavy-handed. It significantly increases the size of the Docker image and makes dependency management more complex. Therefore I'd prefer to keep the building and installation of third party tools separate and up to the user to manage.